go.mod

```diff
@@ -3,47 +3,37 @@ module github.com/CircleCI-Public/circleci-cli
 require (
```

Comment:
The changes in this file are a result of me running `go mod tidy` because I removed the doublestar dependency. It looks like `go mod tidy` is not being run in CI to verify it is updated on every change, so there were more changes than I expected.

Someone may want to add https://circleci.com/orbs/registry/orb/gotest/tools#commands-mod-tidy-check to the CI config